src/plugin/helpers.js

```diff
@@ -108,5 +108,16 @@ export const bulmaComponentList = [
 ]
 
 export function camelCaseToDash (myStr) {
-  return myStr.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase().replace(/([a-z])([0-9])/g, '$1-$2')
+  return myStr
+    .replace(/([a-z])([A-Z])/g, '$1-$2')
+    .toLowerCase()
+    .replace(/([a-z])([0-9])/g, '$1-$2')
+}
+
+export const isBulmaAttribute = attr => !['is-', 'has-', 'fa-'].indexOf(attr)
```

Comment:
that's pretty smart !

Reply:
This is tiny thing, but thank You

Reply:
Guys, I think this is incorrect. At its present state it will only return true if and only if `is-` is the value of `attr`.

Check this:
```
$ node
> let isBulmaAttribute = attr => !['is-', 'has-', 'fa-'].indexOf(attr);
undefined
> isBulmaAttribute('test');
false                                    <----- Ok
> isBulmaAttribute('is-test');
false                                    <----- Not Ok
> isBulmaAttribute('has-test');
false                                    <----- Not Ok
> isBulmaAttribute('fa-test');
false                                    <----- Not Ok
> isBulmaAttribute('is-');
true                                     <----- Technically ok, but useless
> isBulmaAttribute('has-');
false                                    <----- Not Ok
> isBulmaAttribute('fa-');
false                                    <----- Not Ok
```
You may want this instead:
    
    export const isBulmaAttribute = attr => /^(is|has|fa)-/.test(attr);

This is is what it does:
```
$ node
> let isBulmaAttribute = attr => /^(is|has|fa)-.+/.test(attr);
undefined
> isBulmaAttribute('test');
false                                   <----- Ok
> isBulmaAttribute('is-test');
true                                    <----- Ok
> isBulmaAttribute('has-test');
true                                    <----- Ok
> isBulmaAttribute('fa-test');
true                                    <----- Ok
> isBulmaAttribute('is-');
false                                   <----- Ok
> isBulmaAttribute('has-');
false                                   <----- Ok
> isBulmaAttribute('fa-');
false                                   <----- Ok

Reply:
Indeed, after trying it out i do get the same results. Very nice catch here @israelroldan . At first the lib was so simple that i didn't even think about adding tests, but here is a good reason.

I'll add tests then, next PR hopefully today if i have time ! 

In the meanwhile @nandordudas-duna i let you update your PR accordingly.

Reply:
I'm ashamed! I'll fix it on weekends.
@israelroldan: i love regular expressions and your pattern is very useful.
Thank you for your patience and kindness